code/index.html

```diff
@@ -1,22 +1,160 @@
+  <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+  <link rel="manifest" href="site.webmanifest">
+  <!-- FontAwesome Kit Code -->
+  <script src="https://kit.fontawesome.com/819208677a.js" crossorigin="anonymous"></script>
+  <!-- Styles CSS Sheet -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@200;300;400;500;600;700;800;900&family=Raleway:wght@300;400;500;600;700;900&family=Roboto+Condensed:wght@300;400;700&family=Roboto:wght@100;300;400&display=swap"
+    rel="stylesheet">
+  <title>The Future is NOW | News</title>
+</head>
+
+<body>
+
+  <!-- Navbar Section -->
+
+  <header class="navbar">
+    <a class="socials-hidden-1" href="https://twitter.com/tomoxl/status/1473809025254846467" target="_blank" alt="Visit our Twitter"><i class="fab fa-twitter fa-lg social-icons"></i></a>
+    <a class="socials-hidden-2" href="https://www.nytimes.com/2021/10/04/technology/facebook-files.html" target="_blank" alt="Visit our Facebook"><i class="fab fa-facebook fa-lg social-icons"></i></a>
+    <a href="news_site_2/index.html" target="_blank" alt="Home Button"><i class="fas fa-door-open fa-3x"></i></a>
+    <a class="socials-hidden-3" href="https://www.instagram.com/p/CX7zEN_OGWe/" target="_blank" alt="Visit our Instagram"><i class="fab fa-instagram fa-lg social-icons"></i></a>
+    <a class="socials-hidden-4" href="mailto:michaelchangdk@icloud.com" target="_blank" alt="Mail us!"><i class="fas fa-envelope fa-lg social-icons"></i></a>
+    <div class="brand-name">
+      <h1>The Future Is NOW</h1>
+    </div>
+    <div class="socials">
+      <a href="https://twitter.com/tomoxl/status/1473809025254846467" target="_blank" alt="Visit our Twitter"><i class="fab fa-twitter fa-lg social-icons"></i></a>
+      <a href="https://www.nytimes.com/2021/10/04/technology/facebook-files.html" target="_blank" alt="Visit our Facebook"><i class="fab fa-facebook fa-lg social-icons"></i></a>
+      <a href="https://www.instagram.com/p/CX7zEN_OGWe/" target="_blank" alt="Visit our Instagram"><i class="fab fa-instagram fa-lg social-icons"></i></a>
+      <a href="mailto:michaelchangdk@icloud.com" target="_blank" alt="Mail us!"><i class="fas fa-envelope fa-lg social-icons"></i></a>
+    </div>
+  </header>
+
+  <!-- Content Wrapper for all articles -->
+
+  <div class="content-wrapper">
+
+    <!-- Main Article -->
+
+    <div class="head-article">
+      <div class="head-child">
+        <img class="main-image" src="images/spacelaser.jpeg" alt="Space Laser">
+      </div>
+      <div class="head-child">
+        <header class="main-header">
+          <h2>CHINESE RESEARCHER CLAIMS NEW SPACE LASER IS DEFINITELY NOT A WEAPON:<br></h2>
+          <h4>"But a bigger version can be."</h4>
+        </header>
```

Comment:
It would be more easy to understand if you name the classes something like "head-article-img" and "head-article-text" etc.